### PR TITLE
 feat: inplement the add-book data ,remove book data ,saving bookdata…

### PR DESCRIPTION
**Pull request by Kareem24**
… to the local storage and deleting book from the local storage (and in the display UI)

### what?
- Added the add and remove book functionalities
#### why? 
- To add the book to the list and remove it from the list 
#### how?
- I make use of the localStorage built-in function in Java-Script so as to preserve the book data in the browser's local storage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,18 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
     <script src="./js/index.js" defer></script>
-    <link rel="stylesheet" href="./css/index.css" />
+    <title>books</title>
   </head>
-  <body></body>
+  <body>
+    <header><h1>Awesome books</h1></header>
+    <div class="added-books"></div>
```
Comment (Lanr3waju) [inline index.html]: use a semantic tag here

File: index.html [hunk 2]
```diff
@@ -3,9 +3,18 @@
...
+    <form class="book-form" id="form">
+      <!-- <label for="title">title</label> -->
+      <input type="text" name="title" id="title" placeholder="Title" />
+      <br /><br />
```
Comment (Lanr3waju) [inline index.html]: Remove these line breaks, use `CSS` to style your html

File: index.html [hunk 3]
```diff
@@ -3,9 +3,18 @@
...
+      <input type="text" name="author" id="author" placeholder="Author" /><br /><br />
```
Comment (Lanr3waju) [inline index.html]: the input elements should have a label, and if a label is not applicable use, an aria-label for accessibility.

File: jsconfig.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  "compilerOptions": {
+    "module": "ESNext",
+    "moduleResolution": "Node",
+    "target": "ES2020",
+    "jsx": "react",
+    "strictNullChecks": true,
+    "strictFunctionTypes": true
+  },
+  "exclude": ["node_modules", "**/node_modules/*"]
+}
```
Comment (Lanr3waju) [inline jsconfig.json]: Why did you add this file?

**Review by Lanr3waju — COMMENTED** [review]
Great work! But let's make it better, the author and title values should be validated before creating a book with them.

And try to work on and respond to the comments.

File: index.html (hunk 1 above)
Comment (Kareem24) [reply index.html]: Change to an article tag

File: index.html (hunk 2 above)
Comment (Kareem24) [reply index.html]: Done using css

File: jsconfig.json (hunk 4 above)
Comment (Kareem24) [reply jsconfig.json]: **removed**

File: index.html [hunk 5]
```diff
@@ -3,9 +3,21 @@
...
+    <title>books</title>
   </head>
-  <body></body>
+  <body>
+    <section class="book-section">
+      <header><h1>Awesome books</h1></header>
+      <article class="added-books"></article>
+    </section>
+    <form class="book-form" id="form">
+      <!-- <label for="title">title</label> -->
```
Comment (Lanr3waju) [inline index.html]: remove this comment

**Review by Lanr3waju — COMMENTED** [review]
### The validation is not yet good enough, I can add a book when I add only a title and not the author and vice versa, Please correct this.

## Great job!

File: index.html (hunk 5 above)
Comment (Kareem24) [reply index.html]: Removed

**Comment by Kareem24** [comment]
@Lanr3waju thanks for the candid review. I will be making the changes as required and get back to you
